src/pages/api/webhooks/sanity/tips/create.ts

```diff
@@ -0,0 +1,81 @@
+import type {NextApiRequest, NextApiResponse} from 'next'
+import {isValidSignature, SIGNATURE_HEADER_NAME} from '@sanity/webhook'
+import client from '@sanity/client'
+import axios from 'axios'
+import _get from 'lodash/get'
+
+const secret = process.env.SANITY_WEBHOOK_CREATED_SECRET || ''
+const railsToken = process.env.EGGHEAD_ADMIN_TOKEN || ''
```

Comment:
this needed admin permissions on the POST request to rails to create a lesson for any instructor

Reply:
not sure if I could do this another way but I just grabbed an admin access token locally and set that